requirements.txt

```diff
@@ -1,7 +1,7 @@
 numpy>=1.15
 scipy>=1.5.4
 PyYAML
-scikit_learn==0.24.1
+scikit_learn>=0.24.2
```

Comment:
should we just pin this to >= 1.x?

Reply:
Actually, after changing it to 1.0 the py3.6 CI fails. It seems like the latest supported version for that is 0.24.2. @dannygoldstein wachu think?

Reply:
oh womp:
```
Scikit-learn 0.20 was the last version to support Python 2.7 and Python 3.4. scikit-learn 0.23 and later require Python 3.6 or newer. scikit-learn 1.0 and later require Python 3.7 or newer.
```

I see, so we just have to unpin it and let it drift depending on the python version?

Reply:
the client supports 3.6 and since sweeps is an optional dependency of client (i.e., pip install wandb[sweeps]) we should continue supporting 3.6. we can do python version specific requirements, see e.g. https://pip.pypa.io/en/stable/reference/requirements-file-format/#example